Monika After Story/game/script-topics.rpy

```diff
@@ -56,7 +56,48 @@ init 11 python:
 #Use this topic as a template for adding new topics, be sure to delete any
 #fields you don't plan to use
 init 5 python:
-    addEvent(Event(persistent.event_database,eventlabel='monika_god',
+
+addEvent(Event(persistent.event_database,eventlabel='Monika_GLaDOS'    
+                                                            prompt='Monika knowing GlaDOS.',
+                                                            label=None,
+                                                            category=['games'],
+                                                            random=True,
+                                                            unlocked=False,
+                                                            pool=False,
+                                                            conditional=None,
+                                                            action=None,
+                                                            start_date=None,
+                                                            end_date=None,
+                                                            unlock_date=None
+                                                            ))                                              
+label Monika_GLaDOS:                                                
+    m 1a "[player], have you ever played the "Portal series?"
```

Comment:
This is the line with the first double quotes issue I'd recommend:
    m 1a "[player], have you ever played the Portal series?"